### PR TITLE
feat: Section 2 content in nifty JSX

### PR DESCRIPTION
**Pull request by zastrow**
Adds section 2 content and refactors to JSX from MDX.

### CODE REVIEW

File: src/sections/2021/section-2.js [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+import PieChart from '../../components/2021/pie-chart'
+import BarChart from '../../components/2021/bar-chart'
+import Quote from '../../components/2021/quote'
+import Figure from "../../components/2021/figure"
+import SmallCallout from "../../components/2021/small-callout"
+
+const Section2 = () => (
+    <Grid>
+      <GridCell spanMD="8" spanLG="12">
+        <SectionHeader title="Design System & Their Teams" number="2" total="6" />
```
Comment (aspencer116) [inline src/sections/2021/section-2.js:14]: The title should be "The Respondents"

File: src/sections/2021/section-2.js [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+            styleFormat="large"
+            title="Almost 40% of systems were successful or very successful"
+            dataPoints={[
+              ['Not successful', 4],
+              ['Slightly successful', 19],
+              ['Moderately successful', 38],
+              ['Successful', 31],
+              ['Very successful', 8]]
+          }>
+            <p>In your opinion, how <em>successful</em> is your organization’s design system?</p>
```
Comment (aspencer116) [inline src/sections/2021/section-2.js:33]: Emily wants to keep the quote marks on these.

File: src/sections/2021/section-2.js [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+        </Figure>
+      </GridCell>
+      
+      <GridCell>
+        <p>Most in-house respondents felt that their design system is either moderately successful (38%) or successful to very successful (39%).</p>
+      </GridCell>
+      
+      <GridCell>
+        <Quote cite="Executive, successful design system">
+          <p>It has not only outperformed in terms of KPIs, but it has also allowed us to scale more work with fewer people.</p>
```
Comment (aspencer116) [inline src/sections/2021/section-2.js:44]: Add the large body copy class here and `<em><strong>` tags around the bold text from the doc.

File: src/sections/2021/section-2.js [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+            title="Individual contributors usually suggest design systems"
+            dataPoints={[
+              ['An individual contributor(s) suggested starting a design system', 57],
+              ['Leadership suggested starting a design system', 22],
+              ['A third party suggested starting a design system', 3],
+              ['A combination of one or more of the above', 2],
+              ['I am not sure/other', 16]
+            ]}
+          >
+          <p>Where did the <em>idea</em> for your current design system begin?</p>
```
Comment (aspencer116) [inline src/sections/2021/section-2.js:64]: Add quote marks.

File: src/sections/2021/section-2.js [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+        <SmallCallout>
+          <p>16 of the 19 elements were all contained in over 50% of the in-house respondents’ design systems.</p>
+        </SmallCallout>
+      </GridCell>
+      
+      <GridCell>
+        <Figure count="2.4" caption="374 Responses">
+          <BarChart
+            styleFormat="large"
+            title="Organization Size"
```
Comment (aspencer116) [inline src/sections/2021/section-2.js:124]: Should be "Teams are lacking certain disciplines" according to the doc.

File: src/sections/2021/section-2.js [hunk 6]
```diff
@@ -0,0 +1,176 @@
...
+            keyMap={[
+              `Which disciplines do you currently have on your design system team?`,
```
Comment (aspencer116) [inline src/sections/2021/section-2.js:126]: Doc has some red text here. Can we add it?

**Review by aspencer116 — COMMENTED** [review]
Everything looks great I just noticed some copy differences. Not sure if you want to tackle those now or not but figured I'd document them.

File: src/sections/2021/section-2.js (hunk 1 above)
Comment (zastrow) [reply src/sections/2021/section-2.js:14]: Good catch, but “The Respondents” is for section 1. This should be “Characteristics and Teams”

File: src/sections/2021/section-2.js (hunk 3 above)
Comment (zastrow) [reply src/sections/2021/section-2.js:44]: The large body copy will be addressed in #260.

File: src/sections/2021/section-2.js (hunk 6 above)
Comment (zastrow) [reply src/sections/2021/section-2.js:126]: I added in the `<em>`, but they will be printed out as text not html tags for the moment. We need to fix that in a bug card with the component itself.

File: src/sections/2021/section-2.js (hunk 1 above)
Comment (aspencer116) [reply src/sections/2021/section-2.js]: Ah, yes!

**Pull request merged**